### PR TITLE
Pw prime

### PR DESCRIPTION
**Pull request by VikingPaul**
not sure how to tag people in this.. but i guess there's no rush to check it.

### CODE REVIEW

**Comment by jcockhren** [comment]
@VikingPaul tag people like twitter mention 

@sylviama @bernardanderson

File: Mathmagician.Tests/PrimeTest.cs [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.Collections.Generic;
+
+namespace Mathmagician.Tests
+{
+    [TestClass]
+    public class PrimeTest
+    {
+        [TestMethod]
+        public void Prime0()
```
Comment (jcockhren) [inline Mathmagician.Tests/PrimeTest.cs]: @VikingPaul This test needs to be named better. What is it testing? I should know by simply reading the test method name.

File: Mathmagician.Tests/PrimeTest.cs [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        {
+            Assert.IsNotNull(new Prime(1));
+        }
+        [TestMethod]
+        public void Prime1()
```
Comment (jcockhren) [inline Mathmagician.Tests/PrimeTest.cs]: @VikingPaul What is this testing? Name this better.

File: Mathmagician.Tests/PrimeTest.cs [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+        {
+            Assert.IsInstanceOfType(new Prime(1).PrimeList(), typeof(List<int>));
+        }
+        [TestMethod]
+        public void Prime2()
```
Comment (jcockhren) [inline Mathmagician.Tests/PrimeTest.cs]: @VikingPaul What is this testing? Name this better. (again)

File: Mathmagician.Tests/PrimeTest.cs [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+        {
+            Prime num = new Prime(1);
+            Assert.IsTrue(2 == num.PrimeList()[0]);
+        }
+        [TestMethod]
+        public void Prime3()
```
Comment (jcockhren) [inline Mathmagician.Tests/PrimeTest.cs]: @VikingPaul What is this testing? Name this better. (again again)

**Comment by sylviama** [comment]
👍

**Comment by VikingPaul** [comment]
@jcockhren: the tests are named.

File: Mathmagician/Prime.cs [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+namespace Mathmagician
+{
+    public class Prime
+    {
+        List<int> primes = new List<int>();
+        public Prime(int numOfPrimes)
+        {
+            ////////////////////////////primes ALL start with 2 (1 is not a prime number, only has 1 divisor(itself))
+            int lastPrime = 2;
+            Console.WriteLine(numOfPrimes);
```
Comment (jcockhren) [inline Mathmagician/Prime.cs]: @VikingPaul Your Prime class shouldn't not be making a call to the `Console` class. Leave that to Program.cs

**Comment by jcockhren** [comment]
:+1:

**Pull request merged**